packages/jest-core/src/lib/update_global_config.ts

```diff
@@ -68,8 +68,12 @@ export default (
     newConfig.coverageDirectory = options.coverageDirectory;
   }
 
-  if (options.coverageReporters !== undefined) {
-    newConfig.coverageReporters = options.coverageReporters;
```

Comment:
?

Reply:
hm, I didn't notice how removed it, wonder why tests have passed.